### ISSUE TITLE
Major Problem

### ISSUE THREAD

**Issue by ratouney** [EPITECH-Strasbourg-2021/CPP-Arcade-Spec#30]
I have my current `LibNCurses` Class which is inheriting from `Arcade::IGraphicLib`, like this :+1: 
```
class LibNCurses : public Arcade::IGraphicLib {
	public:
		LibNCurses();
		~LibNCurses();
}
```

But when i'm trying to compile after having declared all functions marked as virtual, I'm still having a problem with the destructor, which produces this error : 

```
/tmp/ccvsUt8i.o: In function `Arcade::LibNCurses::LibNCurses()':
source.cpp:(.text+0x83): undefined reference to `Arcade::IGraphicLib::~IGraphicLib()'
/tmp/ccvsUt8i.o: In function `Arcade::LibNCurses::~LibNCurses()':
source.cpp:(.text+0xcc): undefined reference to `Arcade::IGraphicLib::~IGraphicLib()'
collect2: error: ld returned 1 exit status
```
This happens even though I have the declared the destructor for Arcade::LibNCurses.
The problem is that it still somehow requires a destructor to exist for `IGraphicLib`. The only way I've found is to change this : 

```
virtual ~IGraphicLib() = 0;
```

to this :

```
virtual ~IGraphicLib() {};
```

**Comment by ratouney** [comment]
My sources if someone wants to see : 
https://github.com/ratouney/yourcodesucks

**Comment by ratouney** [comment]
@CedricThomas @thibautcornolti will this be updated ?

**Comment by Hirevo** [comment]
This is now addressed in #32.
I'm closing the issue.
